### PR TITLE
Drag pages(s) from one instance to empty instance now possible

### PR DESCRIPTION
**Pull request by kbengs**
Also autoscroll area increased

### CODE REVIEW

**Comment by jeromerobert** [comment]
Thank you ! Your `else` block is almost a copy/past of the `elif target == 'MODEL_ROW_EXTERN'` block which is just above. I think both should be merged.

File: pdfarranger/pdfarranger.py [hunk 1]
```diff
@@ -798,39 +798,41 @@ def iv_dnd_received_data(self, iconview, context, x, y,
             if ref_to:
```
Comment (dreua) [inline pdfarranger/pdfarranger.py]: Is this if clause still useful?

File: pdfarranger/pdfarranger.py [hunk 2]
```diff
@@ -798,39 +798,41 @@ def iv_dnd_received_data(self, iconview, context, x, y,
...
-
+                while data:
+                    tmp = data.pop(0).split('\n')
+                    filename = tmp[0]
+                    npage, angle = [int(k) for k in tmp[1:3]]
+                    crop = [float(side) for side in tmp[3:7]]
+                    pageadder.addpages(filename, npage, angle, crop)
+                if pageadder.commit() and context.get_actions() & Gdk.DragAction.MOVE:
+                    context.finish(True, True, etime)
+
```
Comment (dreua) [inline pdfarranger/pdfarranger.py]: Unnecessary whitespace. I personally can't see it but Codacy can and it's unhappy about it.

File: pdfarranger/pdfarranger.py (hunk 1 above)
Comment (kbengs) [reply pdfarranger/pdfarranger.py]: I tested to remove it and also on line 823 and dragged some pages and I dont see any problem with it. So maybe it does not need to be there..

**Comment by jeromerobert** [comment]
Hello kbengs. Why did you closed that PR ? IMHO it was good.

**Comment by kbengs** [comment]
Hello I found out that it was still a problem with it, when dropping to a a empty instance the pages came in wrong order. So I though I should delete this fast before it gets merged. :)
And I though I would make a PR from #140. Drag to empty instance is also possible there. Have you looked at bengs/pdfarranger/kben/raw fork/branch? Should I make a PR from that?

**Comment by jeromerobert** [comment]
Ok I got it. See my comment on #140.
